indico/modules/events/registration/templates/management/reglist_filter.html

```diff
@@ -97,6 +97,7 @@ <h3>{% trans %}General registration info{% endtrans %}</h3>
         {% set disabled_sections = regform.disabled_sections|selectattr('available_fields')|list %}
         {% if disabled_sections %}
             <h1 class="disabled-caption">
+                {# i18n: Refers to sections in for example registration forms and modals #}
```

Comment:
```suggestion
                {# i18n: Refers to sections in registration forms #}
```

that's not used anywhere else